PodcastGenerator/search.php

```diff
@@ -0,0 +1,66 @@
+<?php
+############################################################
+# PODCAST GENERATOR
+#
+# Created by Alberto Betella and Emil Engler
+# http://www.podcastgenerator.net
+# 
+# This is Free Software released under the GNU/GPL License.
+############################################################
+require 'core/include.php';
+
+// Check for password
+if($config['podcastPassword'] != "") {
+    session_start();
+    if(!isset($_SESSION['password'])) {
+        header('Location: auth.php');
+        die(_('Authentication required'));
+    }
+}
```

Comment:
@emilengler is this how we are checking for authentication?

Reply:
Yes but not the authentication for the admin area. It is the authentication when someone sets a password for the podcast